wrangler-service/src/main/java/co/cask/wrangler/service/connections/ConnectionHandler.java

```diff
@@ -366,16 +358,15 @@ public void updateProp(HttpServiceRequest request, HttpServiceResponder responde
   public void clone(HttpServiceRequest request, HttpServiceResponder responder,
                     @PathParam("id") final String id) {
     try {
-      Connection connection = store.clone(id);
-      if (connection == null) {
-        notFound(responder, String.format(
-          "Connection with id '%s' not found in the store", id
-        ));
-        return;
-      }
-      responder.sendJson(new ServiceResponse<>(ImmutableList.of(connection)));
+      Connection connection = store.get(id);
+      ConnectionMeta clone = ConnectionMeta.builder(connection)
+        .setName(connection.getName() + "_Clone")
```

Comment:
nit: static final?

Reply:
this is only used once, so I think it's more readable to just have it hardcoded here versus one more layer of indirection.